src/sidepacket_servable.cpp

```diff
@@ -80,7 +91,35 @@ SidepacketServable::SidepacketServable(const std::string& modelDir, const std::s
     if (!status.ok()) {
         SPDLOG_ERROR("Error during embeddings node plugin_config option parsing to JSON: {}", pluginConfig);
     }
-    tokenizer = std::make_shared<ov::genai::Tokenizer>(parsedModelsPath);
+    ov::AnyMap tokenizerProperties = {{"add_special_tokens", false}};
+    tokenizer = std::make_shared<ov::genai::Tokenizer>(parsedModelsPath, tokenizerProperties);
+    std::filesystem::path tokenizerConfigPath = (std::filesystem::path(graphPath) / fsModelsPath / "tokenizer_config.json");
+    if (std::filesystem::exists(tokenizerConfigPath)) {
+        std::ifstream ifs(tokenizerConfigPath.string());
+        if (ifs.is_open()) {
+            rapidjson::Document tokenizerConfig;
+            rapidjson::IStreamWrapper isw(ifs);
+            rapidjson::ParseResult parseResult = tokenizerConfig.ParseStream(isw);
+            if (parseResult.Code()) {
+                SPDLOG_ERROR("Parsing tokenizer_config.json failed: {}", rapidjson::GetParseError_En(parseResult.Code()));
+            } else {
+                SET_TOKEN(pad_token);
```

Comment:
isnt there access to such values from ov::genai::Tokenizer object itself? it already reads that config file during tokenizer construction